examples/catalog/src/menu/Menu.tsx

```diff
@@ -20,33 +20,35 @@ export const Menu = () => (
   <MenuContext.Consumer>
     {({isOpen, setOpen, selectedIndex, setSelectedIndex}) => (
       <UserContext.Consumer>
-        {({name, account}) => (
-          <Drawer modal open={isOpen} onClose={() => setOpen(false)}>
-            <DrawerHeader>
-              <DrawerTitle tag='h2'>{name}</DrawerTitle>
-              <DrawerSubtitle>{account}</DrawerSubtitle>
-            </DrawerHeader>
-
-            <DrawerContent>
-              <List singleSelection selectedIndex={selectedIndex}>
-                {Menus.map((menu, index) => (
-                  <ListItem
-                    key={index}
-                    onClick={() => {
+        {({name, account}) => {
+          const close = () => setOpen(false);
```

Comment:
This actually has to be out of the scope of the JSX and render method.

Reply:
But this function used `setOpen`.
And `setOpen` is in the `Context`.
If I move it to out of scope, this method isn't work.